configure.ac

```diff
@@ -174,6 +174,8 @@ if [test $HAVE_SNIFFER = yes]; then
 fi
 
 if [test $IS_FREEBSD = no]; then
+	AC_CHECK_HEADERS([pci/pci.h],,[AC_MSG_ERROR([pciutils header files not found])])
```

Comment:
please, let it be like that:
"pciutils header files not found, consider installing pciutils-devel"